CONTRIBUTING.md

```diff
@@ -23,21 +23,33 @@ git clone https://github.com/{username}/git-cliff && cd git-cliff
 cargo build
 ```
 
-4. Start committing your changes. Follow the [conventional commit specification](https://www.conventionalcommits.org/) while doing so.
+4. Make sure your initial state is successfully prepared by running the tests.
 
-5. Add your tests (if you haven't already) or update the existing tests according to the changes. And check if the tests are passed.
+```sh
+cargo test
+```
+
+If you want to execute tests successfully, you need tags in your repository. The easiest way to do this is to fetch tags from the [git-cliff](https://github.com/orhun/git-cliff) repository.
+
+```sh
+git fetch --tags https://github.com/orhun/git-cliff
+```
```

Comment:
I think we can simply mention running `git fetch --tags` in step 2, after cloning the repository instead of mentioning it separately :)

Reply:
Probably is more reasonable :) I hope the update is ok.

I can also mention without a repository link, but then the upstream must be appropriately handled. I think the link is more straightforward, and copy-paste will be working.